packages/client/package.json

```diff
@@ -82,7 +82,7 @@
     "@prisma/adapter-libsql": "5.4.0",
     "@prisma/adapter-neon": "5.4.0",
     "@prisma/adapter-pg": "5.4.0",
-    "@prisma/adapter-planetscale": "5.4.0",
+    "@prisma/adapter-planetscale": "5.4.1-pre",
```

Comment:
Does "pre" stand for pre-release version?

Reply:
Yes